src/routes/users.ts

```diff
@@ -5,5 +5,6 @@ const users: Router = Router();
 
 users.get('/:id/devices', controller.getDevices);
 users.get('/:id/cats', controller.getCats);
+users.patch('/settings', controller.updateUser);
```

Comment:
You're still not using the auth middleware

Reply:
In the route, add the middleware [like this](https://github.com/clowder-cloud/backend/blob/1b24f8574f26875a9a83c69215e79889bf64cb16/src/routes/cats.ts#L7). You can then replace the first parameter of your controller to the user, [like this](https://github.com/clowder-cloud/backend/blob/1b24f8574f26875a9a83c69215e79889bf64cb16/src/controllers/cats.ts#L7)